Xamarin.Essentials/FilePicker/FilePicker.ios.cs

```diff
@@ -48,6 +48,14 @@ static Task<IEnumerable<FileResult>> PlatformPickAsync(PickOptions options, bool
                 }
             };
 
+            if (documentPicker.PresentationController != null)
+            {
+                documentPicker.PresentationController.Delegate = new PickerPresentationControllerDelegate
+                {
+                    PickHandler = urls => tcs.TrySetResult(Enumerable.Empty<FileResult>())
```

Comment:
Technically, we don't need to do what I am about to type, but maybe for consistency, we should extract the handler from `PickerDelegate.PickHandler ` and then call it here. Maybe a local method might be good?

Reply:
That is an option, however doing it this way is consistent with how it is already done in the `MediaPicker`. Both delegates have their own functionality which they are responsible for.

Reply:
Sounds good.

I am actually looking at the media picker API and I see it doesn't handle exceptions... I'll do a PR on that separately.